### ISSUE TITLE
3.10u dvd drive on a 7000 model not wanting to load

### ISSUE THREAD

**Issue by XxsjrapsxX** [CTurt/FreeDVDBoot#120]
So i had 2 discs made, one for the all slims, since its a slim, and one for english language 3.10 disc drives, for some reason one doesnt show anything, just black, even when left for over 3 minutes, and the other gives me an error saying that the dvd playing function was halted due to malfunction, if you have discord, i could also message you over that, as of right now i havent gotten much of an awnser on this on a youtube video so i decided to go to the source

**Comment by TnA-Plastic** [comment]
Disc-reading issues?

**Comment by XxsjrapsxX** [comment]
> Disc-reading issues?

im not sure, though i dont think so, it runs all disc games perfectly, but when i try to run either the all slims or the one for dvd drives 3.10 and 3.11 it doesnt work. one shows up with nothing and the other shows up with the malfuntion prompt so im ot sure what could be causing it

**Comment by TnA-Plastic** [comment]
Yeah... and there is only ONE thing, which can cause all these kind of issues...
Failing to read the disc properly!

It runs your original discs?

That is even more confirmation that either your discs are trash, or you use a bad disc-writer or the wrong software...

It IS that simple really... Disc-reading-issues!

**Comment by XxsjrapsxX** [comment]
i used the software i was told to use, i used the free iso burner on the lowest speed of an hp dvd-r, so i dont know what you expectfrom me man

**Comment by CTurt** [comment]
I broke the image recently. Just put the old one back, can you please download the latest ISO? Sorry.

**Comment by XxsjrapsxX** [comment]
oh, thanks for the response Cturt, ill go download and try it
